### DISCUSSION TITLE
Is it possible to show Full Path in title but not in Tab?

### DISCUSSION THREAD

**Discussion by nutnutwin** [sumatrapdfreader/sumatrapdf#3149] [Q&A]
# What I know
In ｢SumatraPDF-settings.txt｣
I can set 
> FullPathInTitle = true

so that SumatraPDF would show full path of current name in the Window Title, which is very convenient as I would parse and open the file in fileexplorer

# What I don't understand
Is it possible to show the full path only in Window Title, but not in Tab? 
as the Tab size is limited, it would show only "C:\Users\XXXXX\" and no useful info. 

Thank you very much

**Comment by GitHubRulesOK** [comment]
FullPathInTitle = **False is short name on tabs Title** True = with long path

Its only "Title" when using no tabs (legacy wording)
![image](https://user-images.githubusercontent.com/12754913/204832816-955a7b1c-94c4-4f35-8486-7f5834cfd979.png)

**Comment by nutnutwin** [comment]
Hi, I am sorry for not phrasing properly
![20221201_Thu -124258](https://user-images.githubusercontent.com/34512273/204960878-e06af434-c27d-4c06-be96-7f443b7e1c07.png)

I would love to have

- the ｢Tab｣ on the upper left to show only ｢Temp.pdf｣
- while the whole window shows ｢C:\Temp\Temp.pdf｣

※I am using Autohotkey tools to see what title SumatraPDF is showing. 

Thank you.

**Reply by GitHubRulesOK** [reply]
For AHK I understand Nod5 used to query the file details from the Title Bar so in that case it had to be full path name but depending on task you can export (not query) current file name while tabs are short name. He used a quick trick to get some key data by toggle measure, and possibly that means could be used with file details where full name is first line. 
![image](https://user-images.githubusercontent.com/12754913/204966694-2ce46d7b-3281-4772-be4a-0ba091e175b7.png)

Windows expects the title to follow the same naming as the Tab so here in this discussion Edge will change title bar to show current name. 
![image](https://user-images.githubusercontent.com/12754913/204964331-453ae035-3bee-455e-9628-7014d97a13a1.png)

SumatraPDF is built using similar Microsoft Visual Studio with many component parts following windows conventions

![image](https://user-images.githubusercontent.com/12754913/204964860-0f249db3-9118-473d-a1f3-7fb5ab684335.png)

**Comment by nutnutwin** [comment]
@GitHubRulesOK 
Thank you for your explanation
Now I understand the mechanism
